doc/en/user/source/services/wms/webadmin.rst

```diff
@@ -6,24 +6,31 @@ WMS settings
 This page details the configuration options for WMS in the web administration interface.
 
 .. figure:: img/services_WMS.png
-   
+
    WMS configuration options
 
 Service Metadata
 ----------------
 
-See the section on :ref:`service_metadata`.   
+See the section on :ref:`service_metadata`.
 
 .. _services_webadmin_wms_raster_options:
-   
+
+Root Layer Information
+----------------------
+
+.. figure:: img/services_WMS_rootLayerInfo.png
+
+In this section is possible to define a title and an abstract for the root layer in the WMS capabilities. When these are left empty the WMS service title and abstract are used.
+
```

Comment:
Thanks @jodygarnett, this makes a lot of sense. I would prefer to keep this PR aligned with the Issue, could be done in a separate one?